rust/lon/src/sources.rs

```diff
@@ -16,6 +21,7 @@ const GITHUB_URL: &str = "https://github.com";
 pub struct UpdateSummary {
     pub old_revision: Revision,
     pub new_revision: Revision,
+    pub details: Option<String>,
```

Comment:
Let's be more precise here:

```suggestion
    pub rev_list: Option<String>,
```